bindata/network/ovn-kubernetes/managed/ovnkube-master.yaml

```diff
@@ -756,6 +756,18 @@ spec:
             fi
           done
 
+          #gateway join subnets
+          gateway_v4_join_subnet=
+          gateway_v6_join_subnet=
+          V4JoinSubnet={{.V4JoinSubnet}}
+          V6JoinSubnet={{.V6JoinSubnet}}
+          if [[ -n $V4JoinSubnet ]]; then
+            gateway_v4_join_subnet="--gateway-v4-join-subnet=${V4JoinSubnet}"
```

Comment:
I had thought we were trying to do `ovnkube.conf` options now rather than command-line arguments? Did we change strategies at some point?

Reply:
I am not aware of the strategy. I can refactor the patch.